test/mx/trainer/test_learning_rate_scheduler.py

```diff
@@ -113,18 +113,12 @@ def test_PatientMetricAttentiveScheduler(
         # We construct the scheduler with a different rate, because we need
         # to test that the optimizer can override this setting: this is
         # consistent with how schedulers and optimizers operate in MXNet.
-        base_lr=10 * base_lr,
+        learning_rate=base_lr,
```

Comment:
I guess the comment above does not apply anymore?

Reply:
True.